newrelic/hooks/database_dbapi2_async.py

```diff
@@ -0,0 +1,162 @@
+                return await self.__wrapped__.executemany(sql, seq_of_parameters)
+        else:
+            with DatabaseTrace(
+                sql=sql,
+                dbapi2_module=self._nr_dbapi2_module,
+                connect_params=self._nr_connect_params,
+                cursor_params=self._nr_cursor_params,
+                source=self.__wrapped__.executemany,
+            ):
+                return await self.__wrapped__.executemany(sql, seq_of_parameters)
+
+    async def callproc(self, procname, parameters=DEFAULT):
+        with DatabaseTrace(
+            sql="CALL %s" % procname,
+            dbapi2_module=self._nr_dbapi2_module,
+            connect_params=self._nr_connect_params,
+            source=self.__wrapped__.callproc,
+        ):
+            if parameters is not DEFAULT:
+                return await self.__wrapped__.callproc(procname, parameters)
+            else:
+                return await self.__wrapped__.callproc(procname)
+
+    def __aiter__(self):
+        return self.__wrapped__.__aiter__()
+
+    async def __aenter__(self):
+        await self.__wrapped__.__aenter__()
+        return self
+
+    async def __aexit__(self, exc=None, val=None, tb=None):
+        return await self.__wrapped__.__aexit__(exc, val, tb)
+
+
+class AsyncConnectionWrapper(ObjectProxy):
+
+    __cursor_wrapper__ = AsyncCursorWrapper
+
+    def __init__(self, connection, dbapi2_module, connect_params):
+        super(AsyncConnectionWrapper, self).__init__(connection)
+        self._nr_dbapi2_module = dbapi2_module
+        self._nr_connect_params = connect_params
+
+    def cursor(self, *args, **kwargs):
+        return self.__cursor_wrapper__(
+            self.__wrapped__.cursor(*args, **kwargs), self._nr_dbapi2_module, self._nr_connect_params, (args, kwargs)
+        )
+
+    async def commit(self):
+        with DatabaseTrace(
+            sql="COMMIT",
+            dbapi2_module=self._nr_dbapi2_module,
+            connect_params=self._nr_connect_params,
+            source=self.__wrapped__.commit,
+        ):
+            return await self.__wrapped__.commit()
+
+    async def rollback(self):
+        with DatabaseTrace(
+            sql="ROLLBACK",
+            dbapi2_module=self._nr_dbapi2_module,
+            connect_params=self._nr_connect_params,
+            source=self.__wrapped__.rollback,
+        ):
+            return await self.__wrapped__.rollback()
+
+    async def __aenter__(self):
+        await self.__wrapped__.__aenter__()
+        return self
+
+    async def __aexit__(self, exc=None, val=None, tb=None):
+        return await self.__wrapped__.__aexit__(exc, val, tb)
+
+
+class AsyncConnectionFactory(ObjectProxy):
+
+    __connection_wrapper__ = AsyncConnectionWrapper
+
+    def __init__(self, connect, dbapi2_module):
+        super(AsyncConnectionFactory, self).__init__(connect)
+        self._nr_dbapi2_module = dbapi2_module
+
+    async def __call__(self, *args, **kwargs):
+        rollup = []
```

Comment:
Instead of appending:
```suggestion
        rollup = ["Datastore/all", "Datastore/%s/all" % self._nr_dbapi2_module._nr_database_product]
```